### ISSUE TITLE
[Feature Request] Sort Elements by Stars

### ISSUE THREAD

**Issue by chwzr** [webcomponents/webcomponents.org#1101]
It would be nice if we could sort all Elements by Stars, to see what Components are the most liked ones!

It's not crucial but a good feature for Developers new to Polymer who want to see what Components are mostly used and contributed by the Community.

**Comment by samuelli** [comment]
Yep, some thought would need to go into sorting/filtering since there's been a few feature requests. I have some hesitations around sorting by stars, since its useful to see which have been starred lots, but doesn't necessarily speak to quality. eg. a new component would score low on that list but could still be really good. Stars are still useful, hence why they're shown at all -- I just don't know the true value of sorting by stars.
